content/docs/command-reference/add.md

```diff
@@ -51,8 +51,9 @@ each one:
 5. Add the `targets` to `.gitignore` in order to prevent them from being
    committed to the Git repository (unless `dvc init --no-scm` was used when
    initializing the DVC project).
-6. Instructions are printed showing `git` commands for adding the files, if
-   appropriate.
+6. Instructions are printed showing `git` commands for staging `.dvc` files
+  (or they are staged automatically if
+   [`core.autostage`](/doc/command-reference/config#core) is set).
```

Comment:
This is the perfect place for this note, thanks!

I wonder if other there are similar places in other commands that need this same parenthesis...

Reply:
Please if you see any other places, add this in another PR. Thanks